package.json

```diff
@@ -36,11 +36,15 @@
     "testonly": "npm run tdd -- --single-run"
   },
   "dependencies": {
+    "@react-bootstrap/react-popper": "^1.0.2",
```

Comment:
we'll use the real one when they merge my PR

Reply:
They recently redid the API, and it's essentially what i've have written anyway, so seems like a nice win to use the upstream package, esp. since it's maintained (now) by the popper.js guy

Reply:
great!